src/components/number-picker/number-picker.component.spec.ts

```diff
@@ -10,13 +10,13 @@ import { NumberPickerModule } from './number-picker.module';
                                  [max]="max"
                                  [step]="step"
                                  [valid]="form.controls['integer'].valid"
-                                 [formControl]="form.controls['integer']">
+                                 [formControl]="form.controls['integer']" [placeholder]="placeholder">
```

Comment:
```suggestion
                                 [formControl]="form.controls['integer']"
                                 [placeholder]="placeholder">
```
This should be moved to a new line to match the formatting of the other inputs to improve readability